docker/owlbot/python/cloudbuild.yaml

```diff
@@ -4,7 +4,7 @@ steps:
     entrypoint: '/bin/sh'
     args:
       - '-c'
-      - 'git log -1 --format="%s%n%n%b%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" > post-processor-changes.txt'
+      - 'git log -1 --format="%s%n%nSource-Link: https://github.com/googleapis/synthtool/commit/%H" | sed -e "s/([^()]*)//g" > post-processor-changes.txt'
```

Comment:
This sed statement will remove all parenthesized text:
```
$ echo "Hello (Good Morning) World (#123)" | sed -e "s/([^()]*)//g"
Hello  World
```

Is that acceptable for your purposes?

Reply:
Good catch! I only wanted to remove the parenthesized text at the end. Fixed.